notes/health-care-evaluation.vbs

```diff
@@ -3997,360 +3952,606 @@ If HC_form_name = "No Form - Ex Parte Determination" Then
+			CALL write_bullet_and_variable_in_case_note("* ", "SSI + RSDI")
+			CALL write_bullet_and_variable_in_case_note("* ", "Railroad Retirement Benefits (RRB)")
+			CALL write_bullet_and_variable_in_case_note("* ", "RSDI + RRB")
+			'TO DO - add additional language listing what would qualify for ex parte?
+			'TO DO - add additional case details - case number, renewal info, etc?
+		End If
 
-			Dialog1 = "" 'blanking out dialog name
 
-			BeginDialog Dialog1, 0, 0, 331, 150, "Health Care Renewal Updates - Appears Ex Parte"
-			ButtonGroup ButtonPressed
-				PushButton 205, 130, 100, 15, "Verify HC Renewal Updates", hc_renewal_button
-			Text 5, 5, 320, 10, "Update the following on the Health Care Renewals Screen and then click the button below to verify:"
-			Text 10, 20, 270, 10, "- Elig Renewal Date: Enter one year from the renewal month/year currently listed"
-			Text 10, 35, 100, 10, "- Income/Asset Renewal Date:"
-			Text 25, 45, 290, 20, "- For cases with a spenddown that do not meet an exception listed in EPM 2.3.4.2 MA-ABD Renewals, enter a date six months from the date updated in ELIG Renewal Date"
-			Text 25, 65, 275, 10, "- For all other cases, enter the same date entered in the Elig Renewal Date"
-			Text 10, 80, 145, 10, "- Exempt from 6 Mo IR: Enter N"
-			Text 10, 95, 145, 10, "- ExParte: Enter Y"
-			Text 10, 110, 255, 10, "- ExParte Renewal Month: Enter month and year of the ex parte renewal month"
-			EndDialog
+		'For ex parte denial, write information to case note
+		If ex_parte_determination = "Cannot be Processed as Ex Parte" Then
+			CALL write_variable_in_case_note("Phase 1 - The case has been evaluated for ex parte and has been denied based on the information provided.")
+			CALL write_bullet_and_variable_in_case_note("Reason for Denial:", ex_parte_denial_explanation)
+			'TO DO - add additional language listing what would qualify for ex parte?
+			'TO DO - add additional case details - case number, renewal info, etc?
+		End If
 
+		If ex_parte_determination = "Health Care has been Closed" Then
+			CALL write_variable_in_case_note("No renewal required on case as the Health Care is no longer active.")
+		End If
 
-			DO
-				Do
-					err_msg = ""    'This is the error message handling
-					Dialog Dialog1
-					cancel_confirmation
+		'Add worker signature
+		CALL write_variable_in_case_note("---")
+		CALL write_variable_in_case_note(worker_signature)
 
-					' If ButtonPressed = hc_renewal_button Then Call check_hc_renewal_updates() ' TO DO - timing of function calls and completing function within loop?
+		'Script end procedure
+		script_end_procedure("Success! The ex parte review information has been added to the CASE NOTE")
+	End If
 
-					'TO DO - update with functions?
-					'Check the HC renewal screen data and compare against initial to ensure that changes made properly
+	If ex_parte_phase = "Phase 2" Then
+		Dialog1 = ""
```

Comment:
There should be the functionality to pull the person name in here since it is in the Dialog.

Reply:
Updated - added connection to SQL table to pull in person 1 name and person 2 name (if applicable).

Reply:
This has the ELIG programs still tracking but we never use it. 

Recommend reducing it to something like this:


>       Do While NOT objELIGRecordSet.Eof
> 			If name_01 = "" or name_01 = trim(objELIGRecordSet("Name")) Then
> 				name_01 = trim(objELIGRecordSet("Name"))
> 				PMI_01 = trim(objELIGRecordSet("PMINumber"))
> 			ElseIf name_02 = "" Then
> 				name_02 = trim(objELIGRecordSet("Name"))
> 				PMI_02 = trim(objELIGRecordSet("PMINumber"))
> 			End If
> 			objELIGRecordSet.MoveNext
> 		Loop

Reply:
Updated SQL connection accordingly.